kernel/kernel.c

```diff
@@ -1,15 +1,12 @@
 #include <stdint.h>
+#include <stdio.h>
```

Comment:
Could this clash with the machine stdio.h ? shouldn't it be "stdio.h" or "include/stdio.h" ?

Reply:
I agree, I would go for own header names. `io.h` :P ?  . Same for printf I would go for "info 
, debug, warn, err" names.

Reply:
i guess the name could be ok but the location may clash with the machine where you are building ... afaik <stdio.h> means go to your machine /usr/include to fine the header "stdio.h" means look for it in repo or where you added the "-IXXXX"

Reply:
*find the header

Reply:
it depends you can redefine that in your system, and those bin stuffs just can be used if you compiller is compiled with the flag withsysroot enabled

Reply:
other wise the compiler will try to find in your system